### ISSUE TITLE
Error inflating class error on EditorPreview Android 4.1

### ISSUE THREAD

**Issue by sadach** [imgly/imgly-sdk-android-demo#15]
java.lang.RuntimeException: Unable to start activity ComponentInfo{kr.co.petdoc.petdoc/ly.img.android.ui.activities.PhotoEditorActivity}: android.view.InflateException: Binary XML file line #25: Error inflating class ly.img.android.sdk.views.EditorPreview
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2072)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2097)
	at android.app.ActivityThread.access$600(ActivityThread.java:137)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1203)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:137)
	at android.app.ActivityThread.main(ActivityThread.java:4863)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:511)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:790)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:557)
	at dalvik.system.NativeStart.main(Native Method)
Caused by: android.view.InflateException: Binary XML file line #25: Error inflating class ly.img.android.sdk.views.EditorPreview
	at android.view.LayoutInflater.createView(LayoutInflater.java:613)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:687)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:746)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:749)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:749)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:489)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:396)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:352)
	at com.android.internal.policy.impl.PhoneWindow.setContentView(PhoneWindow.java:321)
	at android.app.Activity.setContentView(Activity.java:2087)
	at ly.img.android.ui.activities.PhotoEditorActivity.onCreate(PhotoEditorActivity.java:70)
	at android.app.Activity.performCreate(Activity.java:5240)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1082)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2036)
	... 11 more
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.constructNative(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:417)
	at android.view.LayoutInflater.createView(LayoutInflater.java:587)
	... 24 more
Caused by: android.view.InflateException: Binary XML file line #36: Error inflating class ly.img.android.sdk.views.LayerContainerView
	at android.view.LayoutInflater.createView(LayoutInflater.java:613)
	at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:687)
	at android.view.LayoutInflater.rInflate(LayoutInflater.java:746)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:459)
	at android.view.LayoutInflater.inflate(LayoutInflater.java:396)
	at ly.img.android.sdk.views.EditorPreview.<init>(EditorPreview.java:106)
	... 27 more
Caused by: java.lang.reflect.InvocationTargetException
	at java.lang.reflect.Constructor.constructNative(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:417)
	at android.view.LayoutInflater.createView(LayoutInflater.java:587)
	... 32 more
Caused by: java.lang.NullPointerException
	at ly.img.android.sdk.views.LayerContainerView.setPadding(LayerContainerView.java:298)
	at android.view.View.<init>(View.java:3673)
	at android.view.ViewGroup.<init>(ViewGroup.java:427)
	at android.widget.RelativeLayout.<init>(RelativeLayout.java:192)
	at ly.img.android.sdk.views.LayerContainerView.<init>(LayerContainerView.java:72)
	at ly.img.android.sdk.views.LayerContainerView.<init>(LayerContainerView.java:68)

**Comment by VladimirSlavic** [comment]
get the same problem, have you figured out why it is?

**Comment by eva-khanh** [comment]
Make sure that you add these line below in your Module build.gradle:
```
renderscriptTargetApi 20
renderscriptSupportModeEnabled true
```

**Comment by eva-khanh** [comment]
me too, please give me a solution!

**Comment by johnteee** [comment]
The same issue on API 16 and above (HTC, Samsung...etc)

**Comment by svendvd** [comment]
Fixed in v2.0.11 RC1
